MekHQ/src/mekhq/campaign/mission/Mission.java

```diff
@@ -274,13 +274,13 @@ public static Mission generateInstanceFromXML(Node wn, Campaign c, Version versi
 
                 if (wn2.getNodeName().equalsIgnoreCase("name")) {
                     retVal.name = wn2.getTextContent();
-                } else if (wn2.getNodeName().equalsIgnoreCase("planetId")) {
-                    retVal.planetId = wn2.getTextContent();
+                } else if (wn2.getNodeName().equalsIgnoreCase("planetId") || wn2.getNodeName().equalsIgnoreCase("systemId")) {
```

Comment:
Looks like this is for loading legacy saves?

Reply:
Yes, this should allow people to seamlessly load in campaigns from before the change. I have tested it on mine and it works fine.